CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ As a result of the changes for **CUMULUS-1193** and **CUMULUS-1264**, **you must
 
 ### BREAKING CHANGES
 
+- **CUMULUS-1228**
+  - The default AMI used by ECS instances is now an NGAP-compliant AMI. This
```

Comment:
So does this mean that if you use a non-NGAP AMI, you can still deploy to a non-NGAP environment?

Reply:
Yes.  I added conditionals to the bootstrap script.